.idea/runConfigurations/templates/LabKeyEmbedded_Dev.xml

```diff
@@ -1,14 +1,15 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="LabKeyEmbbedded_Dev" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
-    <module name="server.server.embedded.main" />
+    <module name="org.labkey.root.server.embedded.main" />
```

Comment:
Is there a better option here so this becomes more generally applicable?

Reply:
I think this is based off of the root project's name in gradle. I have `rootProject.name="labkey-server"` defined in my `settings.gradle` file in several enlistments and they all have `<module name="labkey-server.server.embedded.main" />` for their embedded run configuration.
This also seems to have the same effect for the non-embedded run configurations.
Note that IntelliJ will generate a file (`.idea/.name`) with that name in it, which we would want to ignore or commit.